docs/code-search/code-navigation/precise_code_navigation.mdx

```diff
@@ -52,22 +52,6 @@ Precise Code Navigation requires language-specific indexes to be generated and u
 
 The easiest way to configure precise code navigation is with [auto-indexing](/code-search/code-navigation/auto_indexing). This feature uses [Sourcegraph executors](/admin/executors/) to automatically create indexes for the code, keeping precise code navigation available and up-to-date.
 
-## Precise navigation examples
```

Comment:
Unfortunately, the site generator does not do link checking. Just removing this section will break a bunch of links. We should update the links too.

https://sourcegraph.sourcegraph.com/search?q=context:global+%22precise-navigation-examples%22&patternType=keyword&sm=0